lib/common.js

```diff
@@ -97,6 +97,10 @@ function createConfig(env = {}) {
   const mergeRetries = parsePositiveInt("MERGE_RETRIES", 6);
   const mergeRetrySleep = parsePositiveInt("MERGE_RETRY_SLEEP", 5000);
   const mergeDeleteBranch = env.MERGE_DELETE_BRANCH === "true";
+  const protectBranches = parseMergeRemoveLabels(
```

Comment:
Variable should start with "merge" to indicate it's about merging (not updating). Also, parseMergeRemoveLabels should be renamed then, as it's not really labels which are parsed here

Reply:
Makes sense. I've renamed `protectBranches` to `mergeProtectBranches` and `parseMergeRemoveLabels` to `parseCommaSeparatedList`, which is clearer.